pkg/controllers/raycluster_webhook_test.go

```diff
@@ -46,6 +46,9 @@ func TestRayClusterWebhookDefault(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      rayClusterName,
 			Namespace: namespace,
+			Annotations: map[string]string{
+				versionAnnotation: "0.0.0",
+			},
```

Comment:
Can you remove this annotation from here?
RayCluster CR created by used or SDK won't have the annotation defined. TO reproduce the behavior we shouldn't provide it here either.